manifests/base/tasks/dump-config.yaml

```diff
@@ -70,20 +55,23 @@ spec:
 
         # Check if repo has no commits
         if [ $( git ls-remote --heads $REPO_REMOTE | wc -l ) -eq "0" ]; then
-          echo "Repository empty, creating empty commit"
+          echo "Repository empty, creating empty commit..."
           git init -b $DEFAULT_BRANCH
           git remote add origin $REPO_REMOTE
           git commit --allow-empty -m "Initial commit"
           git push --all origin
         else
+          echo "Repository not empty, cloning..."
           git clone $REPO_REMOTE .
         fi
         git checkout -b $BRANCH_NAME
 
-        cp /mnt/shared/peribolos.yaml .
+        echo "Dumping config..."
+        peribolos --dump $ORG_NAME --github-app-id $APP_ID --github-app-private-key-path /mnt/secret/private_key --confirm > ./peribolos.yaml
+
```

Comment:
Can we keep this as a separate step? It can help with error tracking, logging if peribolos run is separate to repo clonning and PR creation.